docker-compose.yml

```diff
@@ -33,8 +36,8 @@ services:
         source: ./package-lock.json
         target: /opt/csm_web/package-lock.json
         read_only: true
-      # prevent node modules from being overwritten on host
-      - notused:/opt/csm_web/app/node_modules
+    command: sh -c "npm install && npm start"
+
```

Comment:
What's this change for? The entry point should already have done all the setup and server initialization.

(If you revert this, make sure all the changes in this file are reverted.)

Reply:
i think it was changed in the very first base commit fixed now!